### DISCUSSION TITLE
[brew install on macos] Failed to connect to github provider

### DISCUSSION THREAD

**Discussion by AaronGWang** [Homebrew/discussions#5143] [Getting started] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Trying to download brew on macos m2 ventura 13.5 from the original brew.sh website, but failed to connect to github. I tried with and without VPN, different wifi networks, but it doesn't seem to connect. (restarting computer, etc...)

Errror Message: curl: (7) Failed to connect to raw.githubusercontent.com port 443 after 15 ms: Couldn't connect to server

Tried to download brew pkg file, and the installation worked, but brew commands still don't work in terminal. 

Does anyone have a similar issue?

**Comment by SMillerDev** [comment, marked as answer]
> Tried to download brew pkg file, and the installation worked, but brew commands still don't work in terminal.

Did you follow the `shellenv` instructions?
